utkdigitalinitiatives/rfta_canopy
# Ga4

# What Does This Do

This switches UA to GA4 following the same method as other Gatsby things we use.

# How can I test

1. Open deploy below
2. Navigate around
3. Open GA and realtime analytics. Check that things are working as expected.
